packages/node-patches/prettier.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
lets remove this file for now? happy to consider moving the whole repo to prettier in a follow-up